.github/actions/do_build_ock/action.yml

```diff
@@ -0,0 +1,112 @@
+name: build-ock
+description: Action to build the oneapi-construction-kit
+
+# Some of these are riscv or host target specific, but it does not harm at present to
+# overset cmake values
+inputs:
+  build_type:
+    description: 'build type e.g Release, ReleaseAssert (default ReleaseAssert)'
+    default: ReleaseAssert
+  mux_targets_enable:
+    description: 'mux targets to enable e.g. riscv,host (default host)'
+    default: "host"
+  mux_compilers_enable:
+    description: 'mux compiler target to enable e.g. riscv, host (default host)'
+    default: "host"
+  usm:
+    description: 'Enable usm (default ON)'
+    default: ON
+  command_buffer:
+    description: 'Enable command buffers (default ON)'
+    default: ON
+  debug_support:
+    description: 'Enable debug support (default OFF)'
+    default: OFF
+  offline_kernel_tests:
+    description: 'Enable offline kernel testing (default ON)'
+    default: ON
+  llvm_install_dir:
+    description: 'Directory for llvm install'
+    default: ${{ github.workspace }}/llvm_install
+  build_targets:
+    description: 'cmake targets to build (default UnitCL muxc veczc clc)'
+    default: UnitCL muxc veczc clc
+  host_fp16:
+    description: 'Enable host fp16 (default OFF)'
+    default: OFF
+  host_image:
+    description: 'Enable host images (default OFF)'
+    default: OFF
+  host_enable_builtins:
+    description: 'Enable host builtins (default ON)'
+    default: ON
+  external_compiler_dirs:
+    description: 'External compiler directories, default is to empty string, which means no external directories'
+    default: ""
+  hal_description:
+    description: 'Description to be used for HAL, typically risc-v ISA (default RV64GCV)'
+    default: "RV64GCV"
+  hal_refsi_soc:
+    description: 'HAL Refsi SOC e.g. M1 or G1 default M1'
+    default: "M1"
+  riscv_enabled:
+    description: "Enable riscv target (default OFF)"
+    default: OFF
+  build_dir:
+    description: "Directory to be used for building"
+    default: build
+  extra_flags:
+    description: "Any additional cmake flags to be passed (default '')"
+    default:
+  assemble_spirv_ll_lit_test_offline:
```

Comment:
In gitlab we have this OFF on all 'online' builds and ON on 'offline' builds. I fear we'll forget about this option if we don't mimic this behaviour now.

Reply:
I've set it to on for offline

Reply:
Sounds good, but the documented default value below this line is wrong now.